src/ucs/datastruct/array.h

```diff
@@ -390,6 +402,22 @@ ucs_array_old_buffer_set_null(void **old_buffer_p)
     }
 
 
+/**
+ * Modify array capacity (maximum number of elements in the array)
+ *
+ * @param _array        Array whose capacity to set
+ * @param _new_capacity New capacity to set
+ */
+#define ucs_array_set_capacity(_array, _new_capacity) \
```

Comment:
now this macro is used only once so can remove it

Reply:
ok